content/docs/instrumenting/writing_exporters.md

```diff
@@ -461,6 +461,9 @@ putting your username and WIP is fine.
 This is a registry to make our users’ lives a little easier, not a commitment
 to develop particular exporters.
 
+For exporters on internal applications we recommend using ports outside of the
```

Comment:
ok, I still believe some kind of note would be helpful for people in my seat, but I don't care enough to push any further.